src/sentry/models/authenticator.py

```diff
@@ -1,204 +1,3 @@
-                if isinstance(device["binding"], AuthenticatorData):
-                    device["binding"] = base64.b64encode(device["binding"]).decode()
-
-        return super().get_db_prep_value(value, *args, **kwargs)
-
-    def to_python(self, value):
-        ret = super().to_python(value)
-        if self._is_devices_config(ret):
-            for device in ret["devices"]:
-                if isinstance(device["binding"], str):
-                    device["binding"] = AuthenticatorData(base64.b64decode(device["binding"]))
-        return ret
-
-
-@control_silo_model
-class Authenticator(ControlOutboxProducingModel):
-    # It only makes sense to import/export this data when doing a full global backup/restore, so it
-    # lives in the `Global` scope, even though it only depends on the `User` model.
-    __relocation_scope__ = RelocationScope.Global
-
-    id = BoundedAutoField(primary_key=True)
-    user = FlexibleForeignKey("sentry.User", db_index=True)
-    created_at = models.DateTimeField(_("created at"), default=timezone.now)
-    last_used_at = models.DateTimeField(_("last used at"), null=True)
-    type = BoundedPositiveIntegerField(choices=AUTHENTICATOR_CHOICES)
-
-    config = AuthenticatorConfig()
-
-    objects: ClassVar[AuthenticatorManager] = AuthenticatorManager()
-
-    class AlreadyEnrolled(Exception):
-        pass
-
-    class Meta:
-        app_label = "sentry"
-        db_table = "auth_authenticator"
-        verbose_name = _("authenticator")
-        verbose_name_plural = _("authenticators")
-        unique_together = (("user", "type"),)
-
-    def outboxes_for_update(self, shard_identifier: int | None = None) -> list[ControlOutboxBase]:
-        regions = find_regions_for_user(self.user_id)
-        return OutboxCategory.USER_UPDATE.as_control_outboxes(
-            region_names=regions,
-            shard_identifier=self.user_id,
-            object_identifier=self.user_id,
-        )
-
-    @cached_property
-    def interface(self):
-        return AUTHENTICATOR_INTERFACES_BY_TYPE[self.type](self)
-
-    def mark_used(self, save=True):
-        self.last_used_at = timezone.now()
-        if save:
-            self.save()
-
-    def reset_fields(self, save=True):
-        self.created_at = timezone.now()
-        self.last_used_at = None
-        if save:
-            self.save()
-
-    def __repr__(self):
-        return f"<Authenticator user={self.user.email!r} interface={self.interface.interface_id!r}>"
-
-    @classmethod
-    def sanitize_relocation_json(
-        cls, json: Any, sanitizer: Sanitizer, model_name: NormalizedModelName | None = None
-    ) -> None:
-        model_name = get_model_name(cls) if model_name is None else model_name
-        super().sanitize_relocation_json(json, sanitizer, model_name)
-
-        sanitizer.set_string(json, SanitizableField(model_name, "config"), lambda _: '""')
+# flake8 dislikes imported but unused modules
```

Comment:
```suggestion
__all__ = ("Authentictator", )
```

Giving the module an `__all__` is what I've done in the past for these. It also lets you skip the `# NOQA` on the import.